src/Backups/BackupImpl.cpp

```diff
@@ -535,13 +535,15 @@ void BackupImpl::checkBackupDoesntExist() const
     else
         file_name_to_check_existence = ".backup";
 
+    LOG_INFO(&Poco::Logger::get("BackupImpl"), "checkBackupDoesntExist 1");
```

Comment:
Please check unnecessary logging in this PR.

Reply:
addressed it